ironic/dualboot.ipxe

```diff
@@ -0,0 +1,28 @@
+#!ipxe
+
+# NOTE(lucasagomes): Loop over all network devices and boot from
+# the first one capable of booting. For more information see:
+# https://bugs.launchpad.net/ironic/+bug/1504482
+set netid:int32 -1
+:loop
+inc netid || chain pxelinux.cfg/${mac:hexhyp} || goto old_rom
```

Comment:
Shouldn't this be `goto inspector`? Otherwise you'll never get to it. Check the bifrost's file: https://github.com/openstack/bifrost/blob/master/playbooks/roles/bifrost-ironic-install/files/boot.ipxe#L8

Reply:
Also `inc netid` cannot really fail, so the part after `||` is never executed?

Reply:
Ya, I guess we can probably just remove everything after the "||" and all of the old_rom section?

Reply:
Yeah

Reply:
Yeah this is legacy from boot.ipxe in Ironic. Changed.